### DISCUSSION TITLE
Storybook HTML, Show code to show only HTML instead of entire function body?

### DISCUSSION THREAD

**Discussion by ghost** [storybookjs/storybook#12788] [Help] [answered]
In Storybook for HTML, when you click on the Story's "Show code" button, It will expand and show the entire contents of the function instead of the returned HTML string:

![Screen Shot 2020-10-15 at 22 24 54](https://user-images.githubusercontent.com/40529625/96209483-547b9280-0f35-11eb-899b-05138becba32.png)

Is there a way to show only the HTML?

**Comment by shilman** [comment, marked as answer]
coming soon https://github.com/storybookjs/storybook/issues/12755
